tsconfig.json

```diff
@@ -1,6 +1,3 @@
 {
-  "extends": "@tsconfig/node16/tsconfig.json",
-  "compilerOptions": {
-    "esModuleInterop": true
```

Comment:
This is set to true in the base config: https://github.com/tsconfig/bases/blob/be6b3bb160889347b8614e8d18e1e88c40f8ecc9/bases/node20.json#L12